DependencyInjection/HttplugExtension.php

```diff
@@ -145,7 +158,9 @@ private function configurePluginByName($name, Definition $definition, array $con
                 $definition->replaceArgument(0, new Reference($config['cookie_jar']));
                 break;
             case 'decoder':
-                $definition->addArgument($config['use_content_encoding']);
+                $definition->addArgument([
+                    'use_content_encoding' => $config['use_content_encoding'],
```

Comment:
separated the bugfix into #113 and then realized we have other changes in master already, so we won't do a patch release for the fix. but i guess after we merge this we want to tag 1.3 very soon so the bugfix will be released soon.